cmd/xl-storage.go

```diff
@@ -334,6 +335,13 @@ func (s *xlStorage) IsLocal() bool {
 	return true
 }
 
+func (s *xlStorage) Healing() bool {
+	healingFile := pathJoin(s.diskPath, minioMetaBucket,
+		bucketMetaPrefix, healingTrackerFilename)
+	_, err := os.Stat(healingFile)
```

Comment:
Not able to find the code that says that a healing is complete. This looks like it will always return true when the cluster is booted the first time and auto-heal starts to work.

Reply:
Healing is complete in newdisks-heal @vadmeste